File: python/src/2427-commonFactors.py
class Solution:
    def commonFactors(self, a: int, b: int) -> int:
        end = a if a < b else b
        tmp = 0
        for i in range(1, end + 1):
            if a % i == 0 and b % i == 0:
                tmp = tmp + 1
        return tmp

File: js/src/2427-commonFactors.js
/**
 * @param {number} a
 * @param {number} b
 * @return {number}
 */
const commonFactors = function(a, b) {
  const min = Math.min(a, b);
  let tmp = 0;
  for (let i = 1; i <= min; i++) {
    if (a % i === 0 && b % i === 0) {
      tmp++;
    }
  }
  return tmp;
};

export { commonFactors };
